docs/content/config/security/rspamd.md

```diff
@@ -92,6 +92,22 @@ Rspamd provides a [web interface][rspamd-docs::web-interface], which contains st
 
 ![Rspamd Web Interface](https://rspamd.com/img/webui.png)
 
+To use the web interface you will need to configure a password, otherwise you won't be able to log in.
```

Comment:
```suggestion
To use the web interface you will need to configure a password, [otherwise you won't be able to log in][rspamd-docs::web-ui::password].
[rspamd-docs::web-ui::password]: https://www.rspamd.com/doc/tutorials/quickstart.html#setting-the-controller-password
```